cpp/modmesh/mesh/StaticMesh_boundary.cpp

```diff
@@ -122,7 +122,10 @@ void StaticMesh::build_boundary()
 void StaticMesh::build_ghost()
 {
 
-    std::tie(m_ngstnode, m_ngstface, m_ngstcell) = count_ghost();
```

Comment:
I don't know how to unpack and static_cast at the same time.
maybe `std::tuple<size_t,size_t,size_t>(count_ghost())`, but I don't see the benefit